lib/query_relation.rb

```diff
@@ -179,7 +184,7 @@ def dup
   end
 
   def call_query_method(mode)
-    @target.call(mode, options.delete_blanks)
+    @target.call(mode, options.delete_if { |n,v| v.nil? || (v.respond_to?(:empty?) && v.empty?)})
```

Comment:
Really minor but rubocop would probably warn about this if it ran on here

```suggestion
    @target.call(mode, options.delete_if { |_n, v| v.nil? || (v.respond_to?(:empty?) && v.empty?)})
```